src/controllers/routeActivation.ts

```diff
@@ -0,0 +1,105 @@
+import { Context, Next } from 'koa';
```

Comment:
Not sure if this file should be in `controllers` as I feel this acts more like a `middleware`. I'd change the location of this file to `src/middlewares/routeActivation.ts`. Please see if this change is applicable and make the changes

Reply:
Got it will address

Reply:
Addressed